src/mouse/mouse_event.js

```diff
@@ -48,6 +48,18 @@ class MouseEvent {
         this.$pos = this.editor.renderer.screenToTextCoordinates(this.clientX, this.clientY);
         return this.$pos;
     }
+
+    /**
+     * Get the relative position within the gutter.
+     * 
+     * @return {Number} 'row' within the gutter. 
+     */
+    getGutterRow() {
+        var documentRow = this.editor.renderer.screenToTextCoordinates(this.clientX, this.clientY).row;
```

Comment:
Might be better to use `this.getDocumentPosition().row` here

Reply:
ah yeah, good catch, changed